tests/unit/services/infinity-test.js

```diff
@@ -128,6 +128,18 @@ module('Unit | Service | infinity', function(hooks) {
     assert.ok(model instanceof InfinityModel, 'returns cached model again');
   });
 
+  // TODO: set up mirage so a request with `?limit=25` works
+  skip('model instance can be customized by extending InfinityModel', function(assert) {
```

Comment:
@callovarne I think this test and PR is good to go as long as it passes.  Finally took the time to read this pr ;)